Dockerfile

```diff
@@ -22,6 +25,16 @@ RUN --mount=type=cache,target=/root/.cargo/git/db \
     cp target/$(xx-cargo --print-target-triple)/release/typst target/release/typst && \
     xx-verify target/release/typst
 
-FROM alpine:latest
-WORKDIR /root/
+FROM alpine:3.19
+LABEL org.opencontainers.image.authors="Typst <hello@typst.app>"
+LABEL org.opencontainers.image.created=${BUILD_DATE}
+LABEL org.opencontainers.image.description="Typst is an opensource markup-based typesetting system"
+LABEL org.opencontainers.image.documentation="https://typst.app/docs"
+LABEL org.opencontainers.image.licenses="Apache-2.0"
+LABEL org.opencontainers.image.revision=${REVISION}
```

Comment:
Why did you add this but not `org.opencontainers.image.version` which I feel is more relevant?

Reply:
True, but I don't think I have access to that variable in the CI.

Reply:
Don't you do via [`${{ github.event.release.tag_name }}`](https://stackoverflow.com/a/59521156/1567835)? This is run in a release workflow after all. You can add it as a build arg, just like `REVISION`